ember/app/ui/cases/detail/index/template.hbs

```diff
@@ -59,29 +59,25 @@
     </span>
   </div>
   <div class="uk-width-2-3">
-    <h3>
-      {{t "documents.edit"}}
-    </h3>
-    <p>
-      {{t "documents.editText"}}
-    </p>
+    <h3>{{t "documents.edit"}}</h3>
+    <p>{{t "documents.editText"}}</p>
+
     <LinkTo @route="cases.detail.edit" class="uk-button uk-button-default">
       {{t "documents.edit"}}
     </LinkTo>
 
-    <h3>
-      {{t "documents.delete"}}
-    </h3>
-    <p>
-      {{t "documents.deleteText"}}
-    </p>
-    <UkButton
-      @on-click={{perform this.closeCase @model}}
-      @disabled={{this.closeCase.isRunning}}
-      @loading={{this.closeCase.isRunning}}
-    >
-      {{t "documents.delete"}}
-    </UkButton>
+    {{#if (or (eq @model.status "RUNNING") (eq @model.status "SUSPENDED"))}}
```

Comment:
Did you consider using `ember-can` here? If there's more code like this in the future, it might scale better.

Reply:
I am considering adding ember-can to handle authorization. I've never thought about using it to manage other conditions.

But this whole thing will go away anyway as soon as the demo is over and we implement the real UI for mySAGW.

Reply:
Ah ok, then there's probably no need to introduce ember-can.